src/Tizen.Security.DevicePolicyManager/Tizen.Security.DevicePolicyManager/PasswordPolicy.cs

```diff
@@ -148,7 +148,7 @@ public int MinimumRequiredComplexChars
         /// <since_tizen> 6 </since_tizen>
         /// <privilege>http://tizen.org/privilege/dpm.password</privilege>
         /// <privlevel>partner</privlevel>
-        public int MinimunLength
+        public int MinimumLength
```

Comment:
Please add a new Property (MinimumLength) and add Obsolete attribute on current property(MinimunLength)

```suggestion
        [Obsolete("Use MinimumLength instead.")]
        public int MinimunLength => MinimumLength
        public int MinimumLength
```